ConsentViewController/Classes/ConsentViewController.swift

```diff
@@ -84,6 +85,12 @@ import Reachability
     static public let IN_APP_MESSAGING_PAGE_DOMAIN = "in-app-messaging.pm.sourcepoint.mgr.consensu.org/v3/index.html"
 
     static private let MESSAGE_HANDLER_NAME = "JSReceiver"
+    
+    /// The variable is used to store the current SDK version which is used for analytics.
+    static private let CONSENTVIEWCONTROLLER_SDK_VERSION = "3.0.0"
+    
+    /// The varibale used to set the flag to use Rollbar analytics
+    public var isDisableRollbar = false
```

Comment:
I think it'd be better if we name it something a bit more generic (just in case we change service providers in the future). What about `isTelemetryEnabled`? Notice that with this new name, the value is no longer `false` but `true` by default.